modules/rgbd/src/tsdf.cpp

```diff
@@ -434,20 +495,22 @@ void TSDFVolumeCPU::integrate(InputArray _depth, float depthFactor, const cv::Ma
     CV_Assert(_depth.type() == DEPTH_TYPE);
     CV_Assert(!_depth.empty());
     Depth depth = _depth.getMat();
-    IntegrateInvoker ii(*this, depth, intrinsics, cameraPose, depthFactor);
+    Mat pixNorms = preCalculationPixNorm(depth, intrinsics);
+    IntegrateInvoker ii(*this, depth, intrinsics, cameraPose, depthFactor, pixNorms);
     Range range(0, volResolution.x);
-    parallel_for_(range, ii);
+    //parallel_for_(range, ii);
+    ii(range);
```

Comment:
I hope this is a debugging piece of code

Reply:
done